src/Ray.jl

```diff
@@ -14,7 +14,7 @@ using Logging
 using LoggingExtras
 using Pkg
 using Serialization: Serialization, AbstractSerializer, Serializer, deserialize,
-    reset_state, serialize, serialize_type, writeheader
+    reset_state, serialize, serialize_type, ser_version, writeheader
```

Comment:
name is cryptic enough that I think it helps to preface with `Serialization` wherever it is used...

```suggestion
    reset_state, serialize, serialize_type, writeheader
```